popup.html

```diff
@@ -48,10 +48,10 @@
       </div>
 
       <blockquote>
-        <span class="quote">"You could leave life right now. Let that determine what you do and say and think."</span>
-        <span class="quotee">Marcus Aurelius</span>
+        <span class="quote" id="quote">QUOTEHERE</span>
+        <span class="quotee" id="quotee">QUOTEEHERE</span>
```

Comment:
Let's remove `QUOTEEHERE` in case an error happens.

```suggestion
        <span class="quotee" id="quotee"></span>
```